src/app/frontend/error/style.scss

```diff
@@ -17,8 +17,8 @@
 
 .kd-about-logo-icon {
   @include kd-icon-size(8 * $baseline-grid);
-
-  margin: 0 (2 * $baseline-grid) 0 0;
+  margin: 0px;
```

Comment:
`0` doesn't need unit.

Reply:
Yes, sorry. It's a bad habit. I removed that.